creation/web_base/glidein_startup.sh

```diff
@@ -152,1932 +74,467 @@ do_start_all() {
+            if [ -n "${client_repository_group_url}" ]; then
+                echo "GLIDECLIENT_GROUP_WORK_DIR ${client_group_dir}"
+                echo "GLIDECLIENT_DESCRIPTION_GROUP_FILE ${client_descript_group_file}"
+                echo "GLIDECLIENT_Group_Signature ${client_sign_group_id}"
             fi
         fi
+        echo "B64UUENCODE_SOURCE ${PWD}/b64uuencode.source"
+        echo "ADD_CONFIG_LINE_SOURCE ${PWD}/add_config_line.source"
+        echo "GET_ID_SELECTORS_SOURCE ${PWD}/get_id_selectors.source"
+        echo "LOGGING_UTILS_SOURCE ${PWD}/logging_utils.source"
+        echo "GLIDEIN_PATHS_SOURCE ${PWD}/glidein_paths.source"
+        echo "WRAPPER_LIST ${wrapper_list}"
+        echo "SLOTS_LAYOUT ${slots_layout}"
+        # Add a line saying we are still initializing...
+        echo "GLIDEIN_INITIALIZED 0"
+        # ...but be optimist, and leave advertise_only for the actual error handling script
+        echo "GLIDEIN_ADVERTISE_ONLY 0"
+        echo "GLIDEIN_CONDOR_TOKEN ${GLIDEIN_CONDOR_TOKEN}"
+        echo "# --- User Parameters ---"
+    } >> "${glidein_config}"; then
+        early_glidein_failure "Failed in updating '${glidein_config}'"
+    fi
+}
+
+_main(){
+
+    ################################
+    parse_options "$@"
+
+    ################################
+    # Code block used to set the slots_layout
+    # make sure to have a valid slots_layout
+    if (echo "x${slots_layout}" | grep -i fixed) >/dev/null 2>&1 ; then
+        slots_layout="fixed"
+    else
+        slots_layout="partitionable"
     fi
-}
-
-# use this for early failures, when we cannot assume we can write to disk at all
-# too bad we end up with some repeated code, but difficult to do better
-early_glidein_failure() {
-  error_msg="$1"
-
-  warn "${error_msg}"
-
-  sleep "${sleep_time}"
-  # wait a bit in case of error, to reduce lost glideins
-
-  glidein_end_time="$(date +%s)"
-  result="    <metric name=\"failure\" ts=\"$(date --date=@"${glidein_end_time}" +%Y-%m-%dT%H:%M:%S%:z)\" uri=\"local\">WN_RESOURCE</metric>
-    <status>ERROR</status>
-    <detail>
-     ${error_msg}
-    </detail>"
 
-  final_result="$(construct_xml "${result}")"
-  final_result_simple="$(basexml2simplexml "${final_result}")"
-  # have no global section
-  final_result_long="$(simplexml2longxml "${final_result_simple}" "")"
+    ################################
+    parse_arguments
 
-  glidien_cleanup
-
-  print_tail 1 "${final_result_simple}" "${final_result_long}"
-
-  exit 1
-}
-
-
-# use this one once the most basic ops have been done
-glidein_exit() {
-  # Removed lines about $lock_file (lock file for whole machine) not present elsewhere
-
-  gwms_process_scripts "$GWMS_DIR" cleanup "${glidein_config}"
-
-  global_result=""
-  if [ -f otr_outlist.list ]; then
-      global_result=$(cat otr_outlist.list)
-      chmod u+w otr_outlist.list
-  fi
+    ################################
+    # Code block used to generate the glidein UUID
```

Comment:
Similar to functions, it is clear from the file when it is a code block or function. Better to be concise, e.g. 
`# generate the glidein UUID`